test/ibmq/test_basic_server_paths.py

```diff
@@ -47,6 +64,9 @@ def test_job_submission(self):
             with self.subTest(desc=desc, backend=backend):
                 job = self._submit_job_with_retry(ReferenceCircuits.bell(), backend)
 
+                # Ensure job is deleted
+                self.jobs_to_delete.append((provider, job.job_id()))
```

Comment:
Could you move this to the _submit_job_with_retry method so we don't have to add it to every test? You can pass the provider to that method.

Reply:
Updated! For this specifically, the update is [here](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1012/files#diff-762f01fbe80fb731096879850a9910fe15c4286292d868375a0b9fdcbc576c4eR140). A `_sim_job()` function was [also added](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1012/files#diff-11a6c0aad7f62cdf57453b7d159a661b69a91cfb6d7f3e534775f800edc6a4f8R234) to accomplish the same thing for a few other files.